### PR TITLE
[SW-1739] Fix target encoder multiline doc descriptions

### PR DESCRIPTION
**Pull request by jakubhava**
Not causing any problems, but we are adding multiple args instead of one documentation. Should be wrapped in `"""`

### CODE REVIEW

File: py/src/ai/h2o/sparkling/ml/params/H2OTargetEncoderParams.py [hunk 1]
```diff
@@ -61,24 +60,24 @@ class H2OTargetEncoderParams(Params):
     blendedAvgEnabled = Param(
         Params._dummy(),
         "blendedAvgEnabled",
-        "If set, the target average becomes a weighted average of the posterior average for a given "
-        "categorical level and the prior average of the target. The weight is determined by the size "
-        "of the given group that the row belongs to. By default, the blended average is disabled.",
+        """If set, the target average becomes a weighted average of the posterior average for a given
+           categorical level and the prior average of the target. The weight is determined by the size
```
Comment (mn-mikke) [inline py/src/ai/h2o/sparkling/ml/params/H2OTargetEncoderParams.py]: Wouldn't the indentation be included within the text?

Comment (jakubhava) [reply py/src/ai/h2o/sparkling/ml/params/H2OTargetEncoderParams.py]: yes, it probably will, I was following same indentation pattern as above ( keep align to first sentence)

**Comment by jakubhava** [comment]
Oki, spaces will be there. Let me do a quick fix for now. I had a look and there is not a method similar to stripMargin as in scala, extra dependency needs to be done for this. I did this a quick fix without introducing a new dependency

**Comment by jakubhava** [comment]
Created this JIRA https://0xdata.atlassian.net/browse/SW-1742 We can have a look when we have some free time :)

**Pull request merged**